graphite-clickhouse.go

```diff
@@ -59,7 +59,11 @@ func WrapResponseWriter(w http.ResponseWriter) *LogResponseWriter {
 	return &LogResponseWriter{ResponseWriter: w}
 }
 
-func Handler(handler http.Handler) http.Handler {
+type App struct {
+	config *config.Config
+}
+
+func (app *App) Handler(handler http.Handler) http.Handler {
```

Comment:
These are unnecessary changes, right?

Reply:
With app code is simple - no need to pass config as Handler function parameter. But can be removed.